san/tests/test_sanbase_integration.py

```diff
@@ -73,6 +73,25 @@ def test_erc20_projects():
     assert len(result) > 0
     assert len(result[result.slug == "bitcoin"]) == 0
 
+@attr('integration')
+def test_ordinary_function():
+    queries = sanbase_graphql.QUERY_MAPPING.keys()
+
+    for query in queries:
+        if query in ["gas_used", "miners_balance", "mining_pools_distribution"]:
+            slug_to_test = "ethereum"
+        elif query in ["historical_balance", "top_holders_percent_of_total_supply", "social_dominance"]:
+            continue
+        else:
+            slug_to_test = params["project_slug"]
```

Comment:
You can leave this test but IMO explicit > implicit. So it will be better to have multiple functions where you explicitly write down exactly what query is tested. This way you can also break into multiple tests `test_social_data.py`, `test_blockchain_data.py`, `test_prices.py`, etc.

Reply:
You are right, I did as you asked and started making the tests for every function, not one test for all functions